README.md

```diff
@@ -29,40 +29,41 @@ software components we reuse, where and how, as well as their origin, licensing,
 vulnerabilities, quality and other important attributes.
 
 Open source SCA tools are emerging as the best way to help determine which FOSS
-software components are used in a software system or application.
+software components are used in a software system or application. Their output
+can be a software bill-of-material (SBOM).
 
 You are an SCA FOSS tool or project contributor, maintainer, or user?
 
 If so, let's meet at FOSDEM 2021 to share our techniques, experiences and bag of
 tricks and demo or present our FOSS tools to colloborate towards a better SCA
 FOSS toolchain.
 
-
 ### Call for Proposals
 
 We are interested in all kinds of presentations about SCA-related topics using
 Free and Open Source software. Some of the topics include in no particular order:
 
-* Code origin detection techniques
-* Code matching and related code similarity detection
-* Static or dynamic binary analysis for provenance and origin
-* Source code analysis for provenance and origin
-* Package and project metadata collection
-* App and distro dependencies discovery
-* Container and related whole-distro analysis
-* License and copyright detection and analysis
-* Open data for SCA (packages, vulnerabilities, licenses, etc.)
-* Packages and projects quality metrics
+* OSS SCA tools producing a software bill-of-material:
```

Comment:
SCA is not all about SBOM production and consumption. That's only one aspect not the main thing, so I have folded SBOM production and consumption as one topic below and removed these as headers. 
```suggestion
```

Reply:
Yes, but SCA is not about vulnerabilities analysis and consumption I think it can be applied to a number of topics in the call and SBOM is related to SCA similar to license compliance, binary analysis, vulnerability.

Reply:
My objection is to your proposed reorg where all the topics are under two top-level SBOM-related headings of `producing SBOM` and `consuming SBOM`.  These two are fine as topics of their own but not as the top level headings under which everything else would fall e.g. SBOMs are fine tools in support of SCA but they are not central to SCA and the presentation change you made would make SBOM the central points.